spectrum-offchain/src/backlog.rs

```diff
@@ -231,13 +231,20 @@ where
 impl<TOrd, TStore> BacklogService<TOrd, TStore>
 where
     TOrd: OnChainOrder + Weighted + Hash + Eq,
+    TOrd::TOrderId: Debug,
     TStore: BacklogStore<TOrd>,
 {
-    pub fn new<TOrd0: IsEqual<TOrd>>(store: TStore, conf: BacklogConfig) -> Self {
+    pub async fn new<TOrd0: IsEqual<TOrd>>(store: TStore, conf: BacklogConfig) -> Self {
```

Comment:
The main change of this PR. The priority queues in `BacklogService` reside entirely in memory. If the bot ever requires a restart, the elements in the queues are lost.

So on program boot, we inspect the orders that reside in persistent storage and put them back into the `pending_pq` queue. This way we don't 'lose' orders if the bot shuts down.